plugins/jest.js

```diff
@@ -88,6 +88,10 @@ module.exports = {
     // enforce having return statement when testing with promises
     'jest/valid-expect-in-promise': 2,
     // enforce valid titles
-    'jest/valid-title': 2
+    'jest/valid-title': 2,
+    // suggest `await expect(...).resolves` over `expect(await ...)` syntax
+    'jest/prefer-expect-resolves': 2,
+    // suggest using `toBe()` for primitive literals
+    'jest/prefer-to-be': 2
```

Comment:
Can we keep alphabetical ordering plz?

Reply:
For sure, will update the alphabetical order!